tests/core/config/startupFocus.html

```diff
@@ -0,0 +1,30 @@
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<textarea id="editor" contenteditable="true"><p>foo</p></textarea>
+<textarea id="editor_end" contenteditable="true">
+	<p>foo</p>
+	<p>foo</p>
+</textarea>
+<textarea id="editor_table" contenteditable="true"><p>foo</p>
+	<table>
+		<tbody>
+		<tr>
+			<td>cell</td>
+		</tr>
+		</tbody>
+	</table>
+</textarea>
+<textarea id="editor_tailing_bold" contenteditable="true">
+	<p>foo</p>
+	<p>
+		<strong>baz</strong>
+	</p>
+</textarea>
+<textarea id="editor_default" contenteditable="true"><p>foo</p><p>foo</p></textarea>
+<textarea id="editor_start" contenteditable="true"><p>foo</p></textarea>
```

Comment:
I would add one more case with editor containing widget on end to see how it behaves, the content may look like:
```
<p>Paragraph</p>
<figure class="image"><img alt="Saturn V" src="%BASE_PATH%_assets/logo.png" width="200" /></figure>
```
using `image2` plugin (so it have to be added to bender-ckeditor-plugins).